ibm_catalog.json

```
@@ -0,0 +1,605 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-secrets-manager-private-cert-engine",
+        "label": "Cloud automation for Secrets Manager private certificate engine",
```

the terminology used in official [docs](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-prepare-create-certificates) is "private certificates engine" so suggest to update accordingly:
```suggestion
        "label": "Cloud automation for Secrets Manager private certificates engine",
```